CMakeLists.txt

```diff
@@ -1,6 +1,13 @@
 cmake_minimum_required(VERSION 3.4)
 
-project(LibVNCServer VERSION 0.9.14 LANGUAGES C)
+set(PROJECT_LANGUAGES C)
+
+if(NOT ${CMAKE_CXX_COMPILER} STREQUAL "")
```

Comment:
if (DEFINED CMAKE_CXX_COMPILER) is likely better

Reply:
Done. See [abe3462](https://github.com/LibVNC/libvncserver/pull/612/commits/abe34620f6b6571526c400b345a4067fe65edca4).